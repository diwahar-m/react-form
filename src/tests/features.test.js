import { describe, expect, it, test } from "vitest";

// concurrent test 
describe('concurrent test', ()=> {
    it('first test', ()=> {
        expect(2+1).toBe(3);
    })
    it.concurrent('concurrent-1', ({expect})=> {
        expect(2+1).toBe(3);
    })
    it.concurrent('concurrent-2', ({expect})=> {
        expect(2+1).toBe(3)
    })
})

// snapshot 
it("snapshot test", ()=> {
    const upper = toString(12);
    expect(upper).toMatchSnapshot('12');
})

test.skipIf(false)('skiptest', ()=> {
    expect(2+1).toBe(3)
})

test.runIf(true)("only test", ()=> {
    expect(1+1).toBe(2)
})
