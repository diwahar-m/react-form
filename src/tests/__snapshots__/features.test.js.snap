// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot test > 12 1`] = `"[object Undefined]"`;
